lib/plug/router.ex

```diff
@@ -121,9 +121,9 @@ defmodule Plug.Router do
 
   ## Routes compilation
 
-  All routes are compiled to a match function that receives
-  three arguments: the method, the request path split on `/`
-  and the connection. Consider this example:
+  All routes are compiled to a do_match function that receives
```

Comment:
Thank you! The `do_match` function is mostly an implementation detail while the previous match was public. So we need to have a mixture of both approaches. :)